### PR TITLE
Starting work on a code of conduct.

### PR DESCRIPTION
**Pull request by robertmaxrees**
RE: Issue #8.

This is a rough draft - any input would be greatly appreciated.

### CODE REVIEW

**Comment by gnarf** [comment]
I feel this pretty much nails it, but would love input from @ajpiano and @RedWolves

File: pages/irc-help.md [hunk 1]
```diff
@@ -62,6 +62,24 @@ Here are some general points to help you get the most out of your #jQuery experi
...
 
+## Code of Conduct
+
+* Trolling other users isn't tolerated.  We define "trolling" as purposefully inciting arguments for the purposes of angering/harassing other users or derailing the channel.  Disagreements can (and do!) happen, just remember there's a human on the other end.
+
+* Derogatory remarks against any race, religion, lifestyle, orientation, etc. will not be tolerated.
+
+* Kicks, quiets, bans, and other corrective actions are applied at the discretion of whoever has those powers.  If you feel you or somebody else was unfairly treated, please message the op in private.  Airing your grievances in the channel is rarely productive.
+
+* Please refrain from "backseat modding".  If you feel there's an issue that requires intervention, please privately message an op.  You can find a list by running /msg chanserv access list #jquery.
```
Comment (scottgonzalez) [inline pages/irc-help.md]: Let's put the command in backticks.

File: pages/irc-help.md [hunk 2]
```diff
@@ -62,6 +62,24 @@ Here are some general points to help you get the most out of your #jQuery experi
...
+
+* If an op asks you to do something, do it.  If you don't want to, PM the op directly to discuss it.
+
+* Bans and quiets generally expire after 24 hours - sometimes all that's needed is a cool of period.
```
Comment (scottgonzalez) [inline pages/irc-help.md]: typo: cool of

File: pages/irc-help.md [hunk 3]
```diff
@@ -62,6 +62,24 @@ Here are some general points to help you get the most out of your #jQuery experi
 
 * Photographs. Famous plaid-clad people occasionally frequent #jQuery and most will be happy to pose for a photograph with you. Be respectful, say thank you and know when to leave them be.
 
+## Code of Conduct
+
+* Trolling other users isn't tolerated.  We define "trolling" as purposefully inciting arguments for the purposes of angering/harassing other users or derailing the channel.  Disagreements can (and do!) happen, just remember there's a human on the other end.
+
+* Derogatory remarks against any race, religion, lifestyle, orientation, etc. will not be tolerated.
```
Comment (ajpiano) [inline pages/irc-help.md]: perhaps add "nor will stalking, harassment, or ban evasion."

**Comment by scottgonzalez** [comment]
General style nit: One space after periods.

**Comment by robertmaxrees** [comment]
Cleaned up and added extra info per suggestions by @scottgonzalez - thank you for the input!

**Comment by arthurvr** [comment]
To me this looks fine now, any other review or can I merge it?

**Comment by arthurvr** [comment]
Hi @robertmaxrees! Seems like you haven't signed the [CLA](http://contribute.jquery.org/CLA/) yet, could you do so? Make sure the email and name you sign with match your git config.

**Comment by robertmaxrees** [comment]
@arthurvr Good catch!  Re-signed and ready to go.

Thank you!

**Comment by arthurvr** [comment]
Merged. Thanks for working on this, @robertmaxrees!

**Pull request closed without merging**